code/workspaces/infrastructure-api/src/kubernetes/serviceAccountApi.js

```diff
@@ -14,11 +32,13 @@ async function createNamespacedServiceAccount(name, namespace) {
     throw error;
   }
 }
+/* eslint-disable consistent-return */
 
 async function deleteNamespacedServiceAccount(name, namespace) {
```

Comment:
Out of interest, does this succeed if the service account doesn't exist? If not then it might be worth adding the check for it in here as well